client/my-sites/upgrades/domain-management/email/index.jsx

```diff
@@ -41,8 +43,14 @@ const Email = React.createClass( {
 	},
 
 	render() {
+		const showPlanFeatures = config.isEnabled( 'manage/plan-features' );
+
+		const classes = classNames( 'domain-management-email', {
+			'is-wide-layout': showPlanFeatures
```

Comment:
Would you rather the width be set as a style, or we pass in a prop that adds a class?